### PR TITLE
fix(performance-troubleshooting): improve connectors

### PR DESCRIPTION
**Pull request by marielle-spiteri**
* fix links in Connectors section
* Add more details

### CODE REVIEW

File: modules/ROOT/pages/performance-troubleshooting.adoc [hunk 1]
```diff
@@ -108,9 +108,17 @@ that take volume in the database and that slows some queries down.
 
 == Connectors
 
-Any connectors longer than 10 seconds will produce a warning log. See xref:performance-tuning.adoc]#connector_service[connector service] for more information.
+If a connector execution duration exceeds the configured timeout (300 seconds by default), an exception will be raised in the log: 
+[source,log]
+----
+Caused by: org.bonitasoft.engine.connector.exception.SConnectorException: The connector timed out
+---- 
+For Subscription Edition only, this timeout can be increased in the Connector service configuration in `bonita-tenant-community-custom.properties` and `bonita-tenant-sp-custom.properties` (See xref:BonitaBPM_platform_setup.adoc[platform setup]) by increasing the value of the property `bonita.tenant.connector.timeout`.
```
Comment (educhastenier) [inline modules/ROOT/pages/performance-troubleshooting.adoc]:
```suggestion
[NOTE]
====
for subscription editions only.
====
This timeout can be increased in the Connector service configuration in  `bonita-tenant-sp-custom.properties` (See xref:BonitaBPM_platform_setup.adoc[platform setup]) by increasing the value of the property `bonita.tenant.connector.timeout`.  
```

**Review by educhastenier — CHANGES_REQUESTED** [review]
Remove erroneous reference to configuration file bonita-tenant-community-custom.properties

**Comment by marielle-spiteri** [comment]
Thanks @educhastenier , LGTM, feel free to merge 👍

File: modules/ROOT/pages/performance-troubleshooting.adoc [hunk 2]
```diff
@@ -108,9 +108,21 @@ that take volume in the database and that slows some queries down.
...
-
-In addition, performance of connectors can be measured using the xref:performance-tuning.adoc]#time_tracker[connector time tracker].
+If a connector execution duration exceeds the configured timeout (300 seconds by default), an exception will be raised in the log: 
+[source,log]
+----
+Caused by: org.bonitasoft.engine.connector.exception.SConnectorException: The connector timed out
+---- 
+[NOTE]
+====
+for subscription editions only.
```
Comment (tbouffard) [inline modules/ROOT/pages/performance-troubleshooting.adoc]: ⚠️ This is not the wording we use in other pages.

image

or "In Teamwork, Efficiency, Performance and Enterprise editions"

**Review by educhastenier — APPROVED** [review]

**Pull request merged**